webclient/src/forms/RegisterForm/RegisterForm.tsx

```diff
@@ -100,10 +105,10 @@ const RegisterForm = ({ onSubmit }: RegisterFormProps) => {
 
         return (
           <>
-            <form className="RegisterForm" onSubmit={handleSubmit} autoComplete="off">
+            <form className="RegisterForm" onSubmit={handleSubmit}>
               <div className="RegisterForm-column">
                 <div className="RegisterForm-item">
-                  <Field label="Player Name" name="userName" component={InputField} />
+                  <Field label="Player Name" name="userName" component={InputField} autoComplete="username" />
```

Comment:
`autoComplete="username"`?

Reply:
Is this meant to be the `autocomplete="randomString"` hack for Chrome perhaps?

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete

Reply:
Neat!  TIL.